rooms.json5

```diff
@@ -0,0 +1,112 @@
+{
+    "stackexchange.com": {
+        "id": 120914
+
+        "11540": { # Charcoal HQ
+            msg_types: ["debug", "all"],
```

Comment:
These have to be `//`-style comments. If you want `#`-style comments, I’d recommend YAML.